nodes.tf

```diff
@@ -75,6 +75,16 @@ resource "google_container_node_pool" "primary_nodes" {
     image_type = random_id.node_pool_name.keepers.image_type
 
     tags = [var.environment]
+
+    dynamic "kubelet_config" {
```

Comment:
If you know how to play with `dynamic` and `for_each` in terraform, you are an expert! :-)

Reply:
Merit goes to @jahernandezmartinez13 !